api/v1alpha2/bundledeployment_types.go

```diff
@@ -57,6 +57,9 @@ type BundleDeploymentSpec struct {
 	// Config is provisioner specific configurations
 	// +kubebuilder:pruning:PreserveUnknownFields
 	Config runtime.RawExtension `json:"config,omitempty"`
+	// +kubebuilder:validation:Optional
+	// watchNamespaces indicates which namespaces the operator should watch.
+	WatchNamespaces []string `json:"watchNamespaces,omitempty"`
```

Comment:
I think we can create some CEL validation logic here to prevent the `""` value existing in this set using the `// +kubebuilder:validation:XValidation` marker, but I do wonder if we could get away with a combination of the other existing validation markers to ensure the list is unique and make sure that `""` isn't included in the list

Reply:
Fixed with the recent changes.